app/views/layouts/_admin_dropdown.html.erb

```diff
@@ -1,178 +1,40 @@
-<li class="dropdown right">
+<%# Privilegies to check for each menu category %>
+<% anvandare = [["Användare"],[[User], [MailAlias], [Group], [Permission]]] %>
+<% post = [["Poster"], [[Election], [Council], [Document]]] %>
```

Comment:
A hash/array combination would probably be a better choice for this. Like
`anvandare = { 'Användare' => [User, MailAlias, Group, Permission] }`

This would increase readability a lot, see some of my comments below. The helper would also have to be updated, but I think it could be worth it! :)